### PR TITLE
feat(Docs): Docs release notes 4-26-2024

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Added a notice on relevant docs about the October 22, 2024 EOL for containerized private minion and its legacy synthetics runtime versions.
+* Various updates and fixes to our [session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay) docs.
+* When the [Java agent `trim_stats`](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#trim_stats) setting is `true`, a metric is collected when the exclusive time of a segment is less than 2% of the total transaction time.
+* Updated [Java agent classloading issues from JBoss and Wildfly](/docs/apm/agents/java-agent/troubleshooting/classloading-issues-from-jboss-and-wildfly) troubleshooting doc to remove tabs.
+* For [Alerts webhook notification](/docs/alerts-applied-intelligence/notifications/notification-integrations/#set-webhook-destination) integrations, we no longer support OAuth 2.0.
+
+### Release notes and What's New posts
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:49]: Order here seems a bit odd, I'd suggest instead:

* All the WN posts, in alphabetical (or chronological?) order
* All the other release notes, in alphabetical order

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+subject: Docs
+releaseDate: '2024-04-26'
+version: '4-26-2024'
+---
+
+<Callout variant="important">
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:7]: Let's just delete this callout I think

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+</Callout>
+
+### New docs
+
+* Our new [NVIDIA Triton integration](/docs/infrastructure/host-integrations/host-integrations-list/nvidia-triton-integration) monitors the deployment and management of AI models in production environments via the infrastructure agent.
+* Our infrastructure agent Azure monitor integration now supports [Azure Recovery Services Vaults](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-recovery-services-vaults-monitoring-integration).
+
+### Major changes
+
+* A brand new Docs site home page: [check it out](https://docs.newrelic.com/)!
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:18]: Can we add a bit about the logic of why? You can crib from my announcement message in the help channel

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Continued rewrite of on-host integration docs to make it easier to install and ensure the docs are accurate and up to date. Up this week:
+    * [Apache Hadoop](/docs/infrastructure/host-integrations/host-integrations-list/apache-hadoop-monitoring-integration/)
+    * [Apache Zookeeper](docs/infrastructure/host-integrations/host-integrations-list/zookeeper-monitoring-integration/)
+    * [Nextcloud](/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration/)
+    * [NGINX](/install/nginx/)
+    * [Redpanda](/docs/infrastructure/host-integrations/host-integrations-list/redpanda-integration/)
+    * [Sendmail](/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration/)
+    * [VMware VSphere](/install/vsphere/)
+* [Thresholds](/docs/query-your-data/explore-query-data/use-charts/use-your-charts/#thresholds) are a new chart customization that lets our users add useful context to their charts.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:27]:
```suggestion
* [Thresholds](/docs/query-your-data/explore-query-data/use-charts/use-your-charts/#thresholds) are a new chart customization to add useful context to your charts.
```

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+* Updated our [Networks](/docs/new-relic-solutions/get-started/networks) doc to update endpoints and clarify who to reach out to before updating them in future.
+* Updated [browser API `onEnd()`](/docs/browser/new-relic-browser/browser-apis/onend) function syntax.
+* For Kubernetes synthetic monitoring [containerized private minions](/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms), removed Windows as a supported deployment option and clarified that Linux containers can only run on Linux nodes.
+* Added a notice on relevant docs about the October 22, 2024 EOL for containerized private minion and its legacy synthetics runtime versions.
+* Various updates and fixes to our [session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay) docs.
+* When the [Java agent `trim_stats`](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#trim_stats) setting is `true`, a metric is collected when the exclusive time of a segment is less than 2% of the total transaction time.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:45]: These last three seem a bit too weeds-y to be wroth mentioning

**Review by austin-schaefer — DISMISSED** [review]
LGTM, left a few suggestions and comments

File: src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx (hunk 5 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx:45]: Do you mean `four`?

**Pull request merged**